### ISSUE TITLE
Some events non-obviously cause notification count changes

### ISSUE THREAD

**Issue by turt2live** [vector-im/element-meta#1001]
To name a couple:
* Edits: https://github.com/vector-im/element-web/issues/20377
* Poll votes: https://github.com/vector-im/element-web/issues/21115
* Call end events: https://github.com/vector-im/element-web/issues/7941
* Voice Broadcast chunks (actually MSC3245 voice messages)
* Any other new event type we send and don't introduce a push rule for

For reactions and such we fixed this with [MSC2153](https://github.com/matrix-org/matrix-spec-proposals/pull/2153) (a push rule).

This issue exists to track the client-side concern, though the fix is concretely with the spec.

Related: https://github.com/vector-im/element-meta/issues/1088 (possible short-term solution?)

**Comment by SimonBrandner** [comment]
This is sometimes caused by the fact the server can't see the event type due to the event being encrypted (e.g. call hangups)

**Comment by turt2live** [comment]
That just means we're processing push rules either too correctly, or incorrectly. Our behaviour should be exactly the same as in unencrypted rooms.

**Comment by SimonBrandner** [comment]
> That just means we're processing push rules either too correctly, or incorrectly. Our behaviour should be exactly the same as in unencrypted rooms.

We show notif counts based on what the server sends down during sync both in E2EE and non-E2EE rooms which is not ideal :/

**Comment by turt2live** [comment]
That might be an entirely different case then, annoyingly. 

one day we'll fix notifications.

**Comment by HarHarLinks** [comment]
- other call events like switching (my own!) screen sharing state also gives a notification

**Comment by SimonBrandner** [comment]
> * my own!

The other side is responding to your SDP offer with an answer - the two ends need to agree on the media being sent

**Comment by HarHarLinks** [comment]
Thank you for the technical explanation. I'm sure you understand that from an end user perspective it feels like I am the one sending something when I change one of my settings.
Regardless, this should not happen.

**Comment by SimonBrandner** [comment]
> Thank you for the technical explanation. I'm sure you understand that from an end user perspective it feels like I am the one sending something when I change one of my settings. Regardless, this should not happen.

Yes, of course, this is a super annoying (was just trying to add some context as to how this happens)

**Comment by whitequark** [comment]
This has been an issue since 2018 and it's absolutely intolerable in 1:1 voice calls when the other party frequently mutes and unmutes. Are there any concrete plans to fix this?
